index.html

```diff
@@ -1828,11 +1828,62 @@
       </section>
 
       <section>
-        <h3>Bearer Claims</h3>
+        <h3>Bearer Credentials</h3>
 
-        <p class="issue">
-Bearer claims containing PII or unique identifiers can be correlated.
-Bearer claims can be tracked based on usage patterns.
+        <p>
+A <em>bearer credential</em> is a privacy enhancing piece of information,
+such as a concert ticket, that entitles the <a>holder</a> of the claim to a
+particular resource without divulging sensitive information about the holder.
+        </p>
+
+        <p>
+Verifiable Credentials that are bearer credentials are possible by not
+specifying the <a>subject</a> identifier in the <code>claim</code> property.
+For example, the following Verifiable Credential is a bearer credential:
+        </p>
+
+        <pre class="example nohighlight" title="Usage of issuer properties">
+{
+  "id": "http://dmv.example.gov/credentials/temporary/28934792387492384",
+  "type": ["Credential", "ProofOfAgeCredential"],
+  "issuer": "https://dmv.example.gov/issuers/14",
+  "issued": "2017-10-22T12:23:48Z",
+  "claim": {
+    <span class="comment">// note that the 'id' property is not specified for bearer credentials</span>
+    "ageOver": 21
+  },
+  "signature": { ... }
+}
+        </pre>
+
+        <p>
+While bearer credentials can be privacy enhancing, their use must be carefully
+crafted to not accidentally divulge more information than the holder of the
+credential expects. For example, repeated use of the same bearer credential
+across sites enables each site to potentially collude to unduly track or
+correlate the holder. Additionally, information that may seem non-identifying
+such as a birth date and zip code can be used to statistically identify an
+individual when used together in the same credential or session.
+        </p>
+
+        <p>
+<a>Issuers</a> of bearer credentials SHOULD ensure that bearer credentials that
+are expected to provide privacy enhancing benefits 1) are single use, when
+possible, 2) do not contain personally identifying information, and 3) are not
+unduly correlatable.
+        </p>
+
+        <p>
+<a>Holders</a> SHOULD be warned by their software if bearer credentials
+containing sensitive information are issued or requested, or if there is a
+correlation risk when combining two separate bearer credentials across one
+or more sessions. While it may be impossible to detect all correlation risks,
+there are a few simple ones that can be detected.
```

Comment:
Either this should say "some may be detectable" or you should provide the list since it implies that a specific few (and thus nameable) ones can be detected.

Reply:
Fixed, went with the earlier text.